### ISSUE TITLE
Names of positional-only arguments in protocols should be ignored

### ISSUE THREAD

**Issue by Azureblade3808** [microsoft/pylance-release#1187]
## Environment data

-   Language Server version: 2021.4.2
-   OS and version: Windows 7
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.8 Anaconda

## Sample code

```python
from __future__ import annotations

from typing import Protocol

class P(Protocol):
    def f(self, x, /):
        ...

class C:
    def f(self, y):
        ...

c: P = C()  # Should be good, but regarded as incompatible by Pylance.
```

## Expected behavior

The last line is good, as calling `C().f(...)` is valid.

## Actual behavior

There is a warning in the last line.

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.4.3, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202143-29-april-2021
